src/Collapsible.js

```diff
@@ -61,12 +60,14 @@ Collapsible.propTypes = {
    * @default false
    */
   accordion: React.PropTypes.bool,
+  className: React.PropTypes.string,
+  children: React.PropTypes.node,
   /**
    * Enable popout style
    */
   popout: React.PropTypes.bool,
   defaultActiveKey: React.PropTypes.number,
-  onSelect: React.PropTypes.func,
+  onSelect: React.PropTypes.func
```

Comment:
Why don't you like the trailing comma?
It prevents errors when the list is extended, and also avoid introducing changes on the diff :)